als-suggestions/shared/src/main/scala/org/mulesoft/als/suggestions/plugins/aml/webapi/async/Async2SecuredByCompletionPlugin.scala

```diff
@@ -35,7 +30,20 @@ object Async2SecuredByCompletionPlugin extends AMLCompletionPlugin {
     }
   }
 
-  private def getSecurityNames(prefix: String, dp: DeclarationProvider): Seq[RawSuggestion] =
-    new AMLRamlStyleDeclarationsReferences(Seq(SecuritySchemeModel.`type`.head.iri()), prefix, dp, None)
-      .resolve()
+  private def getSecurityNames(request: AmlCompletionRequest): Seq[RawSuggestion] =
+    new AMLRamlStyleDeclarationsReferences(
+          Seq(SecuritySchemeModel.`type`.head.iri()),
+          request.prefix,
+          request.declarationProvider, None
+    ).resolve(rawSuggestionBuilder(request))
+
+  private def rawSuggestionBuilder(request: AmlCompletionRequest)(name: String, de: DomainElement): RawSuggestion = {
+    // todo: aca revisar el DE para la logica de que si tiene scopes es array y sino scalar
```

Comment:
same as above. Try to avoid leaving comments in spanish.

Reply:
the comment still makes sense? or its a leftover?

Reply:
Was a leftover